docs/integrations/github/org.md

```diff
@@ -113,6 +165,10 @@ There is also a `defaultUserTransformer` and `defaultOrganizationTeamTransformer
 You could use these and simply decorate the response from the default
 transformation if you only need to change a few properties.
 
+**Attention:**
+When you use the Events Support with a `TeamTransformer`, you need to ensure your entity will have the
+annotation `github.com/node-id` with the value from `team.id`. Without this, the `team.edit` event could fail to update
```

Comment:
```suggestion
annotation `github.com/team-id` with the value from `team.id`. Without this, the `team.edit` event could fail to update
```

hmm, wouldn't it be clearer for someone inspecting the entity to find team-id instead of node-id?

Reply:
I thought that, but the thing is: We have an `id` for the team and isn't this information. I can change, but this could create some confusion in the future. I will see if the team id could be returned in the graphql query

Reply:
Updated to use the **real** team id :)